doc/ivy.org

```diff
@@ -148,16 +148,20 @@ version that runs Ivy with fancy faces display.
 :CUSTOM_ID: installing-from-emacs-package-manager
 :END:
 
-~M-x~ =package-install= ~RET~ =ivy= ~RET~
+~M-x~ =package-install= ~RET~ =counsel= ~RET~
 
-Ivy is installed as part of =ivy= package, which is available from two
-different package archives, GNU ELPA and MELPA. For the latest stable
-version, use the GNU ELPA archives using the above M-x command.
+Ivy is installed as part of the =counsel= package, which is available from
+two different package archives, GNU ELPA and MELPA. For the latest
+stable version, use the GNU ELPA archives. For current hourly builds,
```

Comment:
Nit: two spaces after the full stop, as per the project's `sentence-end-double-space` setting in `.dir-locals.el`.

Reply:
Thanks for the consistency fix!
I actually noticed the double spaces while editing but found the previous text only had one space, so I tried to conform. :)